### DISCUSSION TITLE
English UI?

### DISCUSSION THREAD

**Discussion by prologic** [swuecho/chat#383] [Q&A] [answered]
HI 👋 Is there an English version of the UI? 🤔

**Comment by swuecho** [comment]
![Screenshot_20230912_161252_com android chrome](https://github.com/swuecho/chat/assets/666683/dc8cde34-9f87-4fa0-a319-a2f1cb469d13)

click the avatar to show the settings

**Reply by prologic** [reply]
Thanks for this. I'm also wondering how can we change the default language on startup?

For example when I start the server and go to http://localhost:8080 I get this (_and I haven't even logged in yet_)

<img width="1712" alt="Screenshot 2023-11-25 at 15 45 33" src="https://github.com/swuecho/chat/assets/1290234/962dc3c8-044e-46b6-b658-6330a6c9d5f6">

**Reply by swuecho** [reply, marked as answer]
This seems to be a bug. The default language should be same with your browser.

**Reply by swuecho** [reply]
fixed. https://github.com/swuecho/chat/pull/391

**Reply by prologic** [reply]
Thanks for this. I had another try and whilst it's somewhat better, there's a still large chunk of the UI that still isn't in English.

**Comment by swuecho** [comment]
> Thanks for this. I had another try and whilst it's somewhat better, there's a still large chunk of the UI that still isn't in English.

most of the Chinese in the message box is prompt. should be fixed now.

Thanks for the feedback.

**Reply by prologic** [reply]
Thank you! 🙇‍♂️
